dart/test/sentry_client_test.dart

```diff
@@ -613,6 +641,38 @@ SentryEvent? eventProcessorDropEvent(SentryEvent event, {dynamic hint}) {
 }
 
 class Fixture {
+  Fixture() {
+    options = SentryOptions(dsn: fakeDsn);
+    options.transport = MockTransport();
+
+    scope = Scope(options)
+      ..user = user
+      ..transaction = transaction
+      ..fingerprint = fingerprint
+      ..addBreadcrumb(crumb);
+
+    event = SentryEvent(
+      level: SentryLevel.warning,
+      transaction: eventTransaction,
+      user: eventUser,
+      fingerprint: eventFingerprint,
+      breadcrumbs: eventCrumbs,
+    );
+  }
+
+  late SentryOptions options;
+  late Scope scope;
+  late SentryEvent event;
+
+  final transaction = '/test/scope';
+  final eventTransaction = '/event/transaction';
+  final fingerprint = ['foo', 'bar', 'baz'];
+  final eventFingerprint = ['123', '456', '798'];
+  final user = SentryUser(id: '123');
+  final eventUser = SentryUser(id: '987');
+  final crumb = Breadcrumb(message: 'bread');
+  final eventCrumbs = [Breadcrumb(message: 'bread')];
```

Comment:
I dont think this should be part of the fixture declaration, this could be specific per test and values can change, I guess?

Reply:
Those won't change though.

Reply:
still, its not part of the fixture pattern, explained a bit more here https://github.com/getsentry/sentry-dart/pull/467/files#r638550478